robot/rover/science/include/Fan.h

```diff
@@ -0,0 +1,25 @@
+//
+// Created by cedric on 2020-10-14.
+//
+
+#ifndef ROVER_FAN_H
+#define ROVER_FAN_H
+
+#include "Updatable.h"
+#include "Stoppable.h"
+
+class Fan :  public Stoppable, public Updatable {
+
+    protected:
+        int fanPWM;
```

Comment:
Since PWM goes from 0-255 why not use uint8 instead of int (which is 2-4 bytes signed)